rad/monadic/issue-remote.rad

```diff
@@ -0,0 +1,183 @@
+;; Issue chain
+;; Meant to be sent to a chain with send-code!
+
+(def chain-id
+  "The name of this chain."
+  "monadic/radicle/issues")
+
+(def issues
+  "Issues are stored in a dict in a ref, keyed by issue-number."
+  (ref {}))
+
+(def issue-counter
+  "We use a stateful counter for giving issues numbers."
+  (lookup :next (make-counter)))
+
+(def used-nonces
+  "We keep track of the UUIDs which have been used as nonces in inputs."
+  (ref set/empty))
+
+(def unused-nonce?
+  "Check that a nonce has not been used already."
+  (fn [nonce]
+    (not (set/member? nonce (read-ref used-nonces)))))
```

Comment:
Btw, I think you can add tests to this file. Just use `:test`, and then in `Tests.hs` run a modified version of it (i.e., one that imports `prelude.rad`)